po/ru.po

```diff
@@ -0,0 +1,2471 @@
+"файлов."
+
+#: lib/results.h:113
+msgid ""
+"Unexpected changed source archive content.  The version of the package did "
+"not change between builds, but the source archive content did.  This may be "
+"deliberate, but needs inspection."
+msgstr ""
+"Неожиданно изменено содержимое исходного архива. Версия пакета не менялась "
+"между сборками, но содержимое исходного архива менялось. Это может быть "
+"сделано намеренно, но требует проверки."
+
+#: lib/results.h:116
+#, c-format
+msgid "Make sure the %%files section includes the %%defattr macro."
+msgstr "Убедитесь, что в разделе %% files есть макрос %% defattr."
+
+#: lib/results.h:117
+msgid ""
+"Bin path files must be owned by the bin_owner set in rpminspect.conf, which "
+"is usually root."
+msgstr ""
+"Файлы пути к папке должны принадлежать bin_owner, установленному в "
+"rpminspect.conf, который обычно является root."
+
+#: lib/results.h:118
+msgid ""
+"Bin path files must be owned by the bin_group set in rpminspect.conf, which "
+"is usually root."
+msgstr ""
+"Файлы пути к папке должны принадлежать группе bin_group, заданной в "
+"rpminspect.conf, обычно это root."
+
+#: lib/results.h:119
+msgid ""
+"Either chgrp the file to the bin_group set in rpminspect.conf or remove the "
+"world execute bit on the file (chmod o-x)."
+msgstr ""
+"Измените chgrp файл в bin_group, установленный в rpminspect.conf, либо "
+"удалите бит выполнения в файле (chmod o-x)."
+
+#: lib/results.h:120
+msgid ""
+"Either chgrp the file to the bin_group set in rpminspect.conf or remove the "
+"group write bit on the file (chmod g-w)."
+msgstr ""
+"Измените chgrp файл в bin_group, установленный в rpminspect.conf, либо "
+"удалите group бит записи в файле (chmod o-x)."
+
+#: lib/results.h:121
+msgid ""
+"Verify the ownership changes are expected. If not, adjust the package build "
+"process to set correct owner and group information."
+msgstr ""
+"Убедитесь, что смена владельца ожидаемая. Если нет, настройте процесс сборки "
+"пакета, чтобы указать правильную информацию о владельце и группе."
+
+#: lib/results.h:124
+msgid "Consult the shell documentation for proper syntax."
+msgstr "Обратитесь к документации оболочки shell для правки синтаксиса."
+
+#: lib/results.h:125
+msgid ""
+"The file referenced was not a known shell script in the before build but is "
+"now a shell script in the after build."
+msgstr ""
+"Указанный файл не был известным сценарием оболочки shell в предыдущей "
+"сборке, но теперь является сценарием оболочки shell в последующей сборке."
+
+#: lib/results.h:126
+msgid ""
+"The referenced shell script is invalid. Consider debugging it with the '-n' "
+"option on the shell to find and fix the problem."
+msgstr ""
+"Указанный shell скрипт недействителен. Рассмотрите возможность отладки с "
+"помощью опции '-n' в терминале, чтобы найти и исправить проблему."
+
+#: lib/results.h:129
+msgid "See annocheck(1) for more information."
+msgstr "См. Дополнительную информацию в annocheck (1)."
```

Comment:
Space between the command and the man page section (1)

Reply:
Done